tests/net/socket/udp/prj.conf

```diff
@@ -10,7 +10,6 @@ CONFIG_NET_SOCKETS=y
 CONFIG_NET_SOCKETS_POSIX_NAMES=y
 
 # Network driver config
-CONFIG_NET_SLIP_TAP=y
```

Comment:
I think the socket tests were not self contained (at least not yet) so they still currently need the SLIP